cpp/src/compaction/stream_compaction_ops.cu

```diff
@@ -78,29 +81,29 @@ private:
 
 typedef repeat_iterator<thrust::detail::normal_iterator<thrust::device_ptr<gdf_valid_type> > > gdf_valid_iterator;
 
-gdf_size_type get_number_of_bytes_for_valid (gdf_size_type column_size) {
-    return sizeof(gdf_valid_type) * (column_size + GDF_VALID_BITSIZE - 1) / GDF_VALID_BITSIZE;
+/**
+ * Calculates the size in bytes of a validity indicator pseudo-column for a given column's size.
+ *
+ * @note Actually, this is the size in bytes of a column of bits, where the individual
+ * bit-container elements are of the same size as `gdf_valid_type`.
+ *
+ * @param[in] column_size the number of elements, i.e. the number of bits to be available
+ * for use, in the column
+ * @return the number of bytes necessary to make available for the validity indicator pseudo-column
+ */
+gdf_size_type get_number_of_bytes_for_valid(gdf_size_type column_size) {
```

Comment:
This looks to be a redefinition of `gdf_get_num_chars_bitmask`. I'm fine with updating the name and implementation, but the definition should be here: https://github.com/rapidsai/cudf/blob/master/cpp/src/utilities/cudf_utils.h#L21

Reply:
The definition already was moved there, and all occurrences were updated.